board-of-directors.php

```diff
@@ -0,0 +1,417 @@
+            <?php if( $image12['title'] ): ?>
+              <p class="title"><?php echo $image12['title'] ?></p>
+            <?php endif; ?>
+        </div>
+      </div>
+   </div>
+   <?php endif; ?>
+</div>
+
+
+<div class="row">
+<?php if( !empty($image13)): ?>
+    <div class="col-xs">
+      <div class="hovereffect">
+        <img class="large-img" src="<?php echo $image13['sizes'][ $size ]; ?>" alt="<?php echo $image13['alt']; ?>"?>
+        <?php if( $image13['title'] ): ?>
+        <p class="title"><?php echo $image13['title'] ?></p>
+        <?php endif; ?>
+        <div class="overlay">
+          <div class="content">
+            <img class="small-img" src="<?php echo $image13['sizes'][ $size ]; ?>">
+            <button type="submit" href="#">Learn More</button>
+            <?php if( $image13['caption'] ): ?>
+              <p><?php echo $image13['caption'] ?></p>
+            <?php endif; ?>
+          </div>
+            <?php if( $image13['title'] ): ?>
+              <p class="title"><?php echo $image13['title'] ?></p>
+            <?php endif; ?>
+        </div>
+      </div>
+    </div>
+    <?php endif; ?>
+
+<?php if( !empty($image14)): ?>
+    <div class="col-xs">
+      <div class="hovereffect">
+        <img class="large-img" src="<?php echo $image14['sizes'][ $size ]; ?>" alt="<?php echo $image14['alt']; ?>"?>
+        <?php if( $image14['title'] ): ?>
+        <p class="title"><?php echo $image14['title'] ?></p>
+        <?php endif; ?>
+        <div class="overlay">
+          <div class="content">
+            <img class="small-img" src="<?php echo $image14['sizes'][ $size ]; ?>">
+            <button type="submit" href="#">Learn More</button>
+            <?php if( $image14['caption'] ): ?>
+              <p><?php echo $image14['caption'] ?></p>
+            <?php endif; ?>
+          </div>
+            <?php if( $image14['title'] ): ?>
+              <p class="title"><?php echo $image14['title'] ?></p>
+            <?php endif; ?>
+        </div>
+      </div>
+    </div>
+    <?php endif; ?>
+
+
+<?php if( !empty($image15)): ?>
+    <div class="col-xs">
+      <div class="hovereffect">
+        <img class="large-img" src="<?php echo $image15['sizes'][ $size ]; ?>" alt="<?php echo $image15['alt']; ?>"?>
+        <?php if( $image15['title'] ): ?>
+        <p class="title"><?php echo $image15['title'] ?></p>
+        <?php endif; ?>
+        <div class="overlay">
+          <div class="content">
+            <img class="small-img" src="<?php echo $image15['sizes'][ $size ]; ?>">
+            <button type="submit" href="#">Learn More</button>
+            <?php if( $image15['caption'] ): ?>
+              <p><?php echo $image15['caption'] ?></p>
+            <?php endif; ?>
+          </div>
+            <?php if( $image15['title'] ): ?>
+              <p class="title"><?php echo $image15['title'] ?></p>
+            <?php endif; ?>
+        </div>
+      </div>
+   </div>
+   <?php endif; ?>
+</div>
+
```

Comment:
Looks like all of these are the same blocks of code, can you instead make a for loop and loop through all of the image ids?